shell/packages/accounts-email-token/token_server.js

```diff
@@ -53,8 +57,9 @@ Accounts.registerLoginHandler("emailToken", function (options) {
     };
   }
 
-  if (!user.services || !user.services.emailToken ||
-      !(user.services.emailToken.tokens)) {
+  var identity = user.identities[0];
```

Comment:
Thanks for that link.  I tried a bunch of variations of google searches on "mongodb $ field" and never managed to get to that page.